synfig-studio/src/gui/renddesc.cpp

```diff
@@ -151,7 +154,7 @@ Widget_RendDesc::refresh()
 	entry_duration->set_value(rend_desc_.get_duration());
 
 	adjustment_fps->set_value(rend_desc_.get_frame_rate());
-
+	//zzzzzzzzzzzzzzzzzzzzz
```

Comment:
I suggest to remove this line. :satisfied:

Reply:
That's my way to mark things as important 😂